h2quic/client_test.go

```diff
@@ -4,97 +4,102 @@ import (
 	"bytes"
 	"compress/gzip"
 	"errors"
+	"net"
 	"net/http"
 
 	"golang.org/x/net/http2"
 	"golang.org/x/net/http2/hpack"
 
+	quic "github.com/lucas-clemente/quic-go"
```

Comment:
Maybe you mean that it would be sufficient to just write `"github.com/lucas-clemente/quic-go"` instead of `quic "github.com/lucas-clemente/quic-go"`. That's true, but `gofmt` always adds the `quic`, so I'd rather leave it in here.